.github/workflows/podvm_binaries.yaml

```diff
@@ -57,12 +57,31 @@ jobs:
         username: ${{ github.repository_owner }}
         password: ${{ secrets.GITHUB_TOKEN }}
 
+    - name: Determine image tags
```

Comment:
Shouldn't the `GITHUB_ENV` be passed across downstream jobs? If I understand it correctly then this step should be only present in `podvm_builder` and the remaining steps should simply rely on it (I'd like to avoid duplicating the embedded steps)

Reply:
Hmm, I can see later that you perhaps want these to be callable separately. Not sure if it's a benefit as it need quite a lot of code-duplication. Anyway let me know what you think about the `GITHUB_ENV` or `INPUTS` use for these as the steps depend on each-other anyway so I don't think they are likely to be called with different arguments.

Reply:
hi @ldoktor ! First of all, thanks for reviewing this PR.

I did not understand your idea, though.

Sending a variable to `$GITHUB_ENV` ensures the next steps within same job will have the variable exported on the environment.

In this PR, I made the `podvm_builder`, `podvm_binaries` and `podvm` workflows callable so they can be used by different workflows. And the way to passes values to those callable workflows is via `inputs`. If there is a way o inherent environment variables between jobs I am not aware of.

Reply:
I'm not sure if `GITHUB_ENV` could be passed between jobs so you are most certainly right. But the `INPUTS` should work. Not sure if it would result in less boiler-plate, though.

Reply:
I mean it's not a deal-breaker, I just dislike the amount of repeated work. If we constructed the `CAA_SRC_REF` in the first workflow and passed only the product, we might remove most of the duplicities.